package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "remark-language-server",
+  "version": "1.0.0",
+  "description": "Language server for markdown using remark",
+  "homepage": "https://remark.js.org",
+  "repository": "https://github.com/remarkjs/remark/tree/main/packages/remark-language-server",
+  "bugs": "https://github.com/remarkjs/remark-language-serve/issues",
+  "bin": "index.js",
+  "author": "Remco Haszing <remcohaszing@gmail.com>",
+  "license": "MIT",
+  "type": "module",
+  "files": [
+    "index.js"
+  ],
+  "keywords": [
```

Comment:
Some of the order seems random here. See an example of the order: https://github.com/unifiedjs/unified-language-server/blob/fc82fbe4df16c25480861e16951b9840bfbe2b30/package.json#L6

Reply:
Do you mean it should be alphabetized? That would be my guess, but you’re referencing a place where keywords are also not alphabetized.

Reply:
I don’t mean alphabetized, I mean ordered differently. That order is: metadata, files/exports, deps, scripts, extra stuff.
Another example: https://github.com/unifiedjs/unified-engine/blob/main/package.json

Reply:
Also: I mean the props themselves, not the keywords, sorry if that was confusing

Reply:
You may want to check out [prettier-plugin-packagejson](https://github.com/matzkoh/prettier-plugin-packagejson) if you want to enforce this within unified. Personally I like it for mono repos, but for single packages I think it’s a bit overkill.

Anyway: done

Reply:
Yeah, same. There’s a vague line between fast and simple installs and development, and useful and powerful and expansive tooling.
I’m pushing for consistency here based on other repos.
Similarly, a JSDoc plugin might also be useful, but it also feels a bit like “too much”. ⚖️